src/solver/solve.jl

```diff
@@ -243,6 +242,10 @@ function check_conservation(sol)
     end
     energy_radiation_input = -p.net_energy_flux_toa[][]
 
+    @info energy_atmos_change,
```

Comment:
This block prints as follows: 
```
[ Info: Checking conservation
[ Info: (-3.5128077f17, -0.0f0, -0.0f0, -9.468598f22)
[ Info:     Net energy change / total energy: -3.7099553e-6
[ Info:     Net mass change / total mass: 1.2692552e-6
[ Info:     Net water change / total water: 4.2512484e-6
```
The 4-tuple values are not really clear from just the logging statement so maybe we can add string descriptions (or remove this info block entirely, since `driver.jl` prints the change after the check is complete). I checked that the `energy, mass` txt files are generated at the end of a baroclinic wave test simulation.

Reply:
Oh, I forgot to remove this. I added them to check the diagnostics are correct. I'll remove them.